packages/scanner/bin/build.js

```diff
@@ -0,0 +1,29 @@
+const { lstatSync, readdirSync, copyFileSync, existsSync, mkdirSync } = require('fs');
+const path = require('path');
+
+function copyFolderRecursively(source, destination) {
+  let files = [];
+  const targetFolder = path.join(destination, path.basename(source));
+
+  if (!existsSync(targetFolder)) {
+    mkdirSync(targetFolder);
+  }
+
+  if (lstatSync(source).isDirectory()) {
+    files = readdirSync(source);
+
+    files.forEach((file) => {
+      const currentSource = path.join(source, file);
+      const currentDestination = path.join(targetFolder, path.basename(file));
+
+      if (lstatSync(currentSource).isDirectory()) {
+        copyFolderRecursively(currentSource, targetFolder);
+      } else {
+        copyFileSync(currentSource, currentDestination);
+      }
+    });
+  }
+}
+
+mkdirSync('built', { recursive: true });
+copyFolderRecursively('src/sampleConfig', 'built');
```

Comment:
The package already pulls `fs-extra` as a dev dependency, might as well do `fsextra.copy('src/sampleConfig', 'built/sampleConfig')` instead of rolling your own. 

BTW, the name of this file is misleading — it doesn't build anything, it just prepares the output directory.